Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM gableroux/flutter:v1.7.6
+
+WORKDIR /app
+
+COPY ./app_feup/ .
+
+RUN flutter pub get
+
+CMD [ "bash", "-c" , "flutter analyze --no-pub --preamble . && flutter test"]
```

Comment:
This exists but travis does not use it 🙃 
Was this on purpose?

Also, I'm not sure if docker run with a command overrides the `CMD` given here or if it will just always run both.

Furthermore, it might make sense to rename this file to `Dockerfile-test`, given its scope.

Reply:
Also, please change `bash -c` to `sh -c`. It's mostly irrelevant but still a good practice.

Also, you could just specify the command in `CMD` - I'm guessing this was laziness because of the syntax, because otherwise it does not make much sense to me to be like this.

Reply:
It was. This way everyone can build and run it on docker on their machine if they so wish to without having the need to know the flags of flutter and all.

I had that doubt myself and checked. It does not.

Well I intend to further build into this dockerfile in the future to try and be able to deploy changes of the app to automatically deploy to App stores, that's mainly why I didn't change it

It was not laziness at all, I did try to run the other two commands separating the strings. How would you approach this then? Would you use a shell script?